### ISSUE TITLE
Indent Guide Coloring

### ISSUE THREAD

**Issue by sooxt98** [microsoft/vscode#130379]
Is it possible to archive this? I saw both intellij and vs2017 are able to archive it with plugins
![image](https://user-images.githubusercontent.com/13378059/128654832-c481d97a-a0a9-4f14-bdc4-f7ca39bd67d3.png)
![image](https://user-images.githubusercontent.com/13378059/128654835-b5b789d3-bbab-4fec-b287-d413bbf959c0.png)

**Comment by gjsjohnmurray** [comment]
pinging @hediet because of #129231

**Comment by hediet** [comment]
I will explore this after #129231 landed.

However, I'm sure there are some extensions that do this (although they might be slow)!

**Comment by ArturoDent** [comment]
Some info here: https://stackoverflow.com/questions/47942390/how-to-change-indent-guide-line-color-between-brackets-in-vscode

I am sure I have seen an extension that does what you seek but I can't find it now.

**Comment by jjspace** [comment]
I use `indent-rainbow` to acheive a similar effect, it's not the guides themselves but it still gives each indent level a color which is helpful https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow

**Comment by thoroc** [comment]
Would be nice to see the functionality of the extension folded into vscode proper. one less extension to load.

**Comment by smdsbz** [comment]
I currently use settings for the bracket pair to achieve such visual. However, I found the colors too distracting and configured it so it only shows no color, only contrast 😂 
![image](https://user-images.githubusercontent.com/19503270/169761822-85ba146a-6fe1-42cf-af46-e60a0993fe74.png)
Which gives me such result
![image](https://user-images.githubusercontent.com/19503270/169762065-afd9d015-b8dc-44eb-9974-40ee39fddff2.png)

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for [similar existing issues](https://github.com/microsoft/vscode/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+). See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Happy Coding!

**Comment by 0x5c** [comment]
What is the issue this one is a duplicate of?

**Comment by hediet** [comment]
These are bracket pair guides.
